tests/config/test_schemas.py

```diff
@@ -83,7 +83,7 @@ def test_deserialize_deprecated_value(self):
         self.schema['foo'] = types.Deprecated()
 
         result, errors = self.schema.deserialize(self.values)
-        self.assertItemsEqual(['bar', 'baz'], result.keys())
+        self.assertEqual(['bar', 'baz'], list(result.keys()))
```

Comment:
```self.assertEqual(sorted(['bar', 'baz']), sorted(result.keys()))``` ?

Reply:
Ah, no, the problem here wasn't ordering. `assertItemsEqual` does not exist in Py3.7, and `result.keys()` returns a dict view, not a list. This was the minimum set of changes to make it pass.